[Bug]: Webpack examples are not working

### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

webpack examples

### Expected behavior

Building the webpack examples should produce a bundle usable in the browser

### Actual behavior

Webpack build ends up with errors:

ERROR in ./src/index.js 17:0-51
Module not found: Error: Package path ./sp-button is not exported from package /Users/livvius/IdeaProjects/playground/spectrum-web-components/projects/example-project-webpack/node_modules/@spectrum-web-components/button (see exports field in /Users/livvius/IdeaProjects/playground/spectrum-web-components/projects/example-project-webpack/node_modules/@spectrum-web-components/button/package.json)

ERROR in ./src/index.js 18:0-61
Module not found: Error: Package path ./sp-field-label is not exported from package /Users/livvius/IdeaProjects/playground/spectrum-web-components/projects/example-project-webpack/node_modules/@spectrum-web-components/field-label (see exports field in /Users/livvius/IdeaProjects/playground/spectrum-web-components/projects/example-project-webpack/node_modules/@spectrum-web-components/field-label/package.json)

ERROR in ./src/index.js 19:0-51
Module not found: Error: Package path ./sp-picker is not exported from package /Users/livvius/IdeaProjects/playground/spectrum-web-components/projects/example-project-webpack/node_modules/@spectrum-web-components/picker (see exports field in /Users/livvius/IdeaProjects/playground/spectrum-web-components/projects/example-project-webpack/node_modules/@spectrum-web-components/picker/package.json)

ERROR in ./src/index.js 20:0-47
Module not found: Error: Package path ./sp-menu is not exported from package /Users/livvius/IdeaProjects/playground/spectrum-web-components/projects/example-project-webpack/node_modules/@spectrum-web-components/menu (see exports field in /Users/livvius/IdeaProjects/playground/spectrum-web-components/projects/example-project-webpack/node_modules/@spectrum-web-components/menu/package.json)

ERROR in ./src/index.js 21:0-52
Module not found: Error: Package path ./sp-menu-item is not exported from package /Users/livvius/IdeaProjects/playground/spectrum-web-components/projects/example-project-webpack/node_modules/@spectrum-web-components/menu (see exports field in /Users/livvius/IdeaProjects/playground/spectrum-web-components/projects/example-project-webpack/node_modules/@spectrum-web-components/menu/package.json)

### Screenshots

_No response_

### What browsers are you seeing the problem in?

_No response_

### How can we reproduce this issue?

_No response_

### Sample code that illustrates the problem

_No response_

### Logs taken while reproducing problem

_No response_

--- a/projects/example-project-webpack/src/index.html
+++ b/projects/example-project-webpack/src/index.html
@@ -15,25 +15,27 @@
     </head>
 
     <body>
-        <sp-button variant="primary">Primary</sp-button>
-        <br />
-        <br />
-        <sp-button variant="secondary" quiet>secondary</sp-button>
-        <br />
-        <br />
-        <sp-field-label for="picker">Where do you live?</sp-field-label>
-        <sp-picker id="picker">
-            <span slot="label">
-                Select a Country with a very long label, too long in fact
-            </span>
-            <sp-menu-item>Deselect</sp-menu-item>
-            <sp-menu-item>Select Inverse</sp-menu-item>
-            <sp-menu-item>Feather...</sp-menu-item>
-            <sp-menu-item>Select and Mask...</sp-menu-item>
-            <sp-menu-divider></sp-menu-divider>
-            <sp-menu-item>Save Selection</sp-menu-item>
-            <sp-menu-item disabled>Make Work Path</sp-menu-item>
-        </sp-picker>
+        <sp-theme scale="medium" color="light">
+            <sp-button variant="primary">Primary</sp-button>
+            <br />
+            <br />
+            <sp-button variant="secondary" quiet>secondary</sp-button>
+            <br />
+            <br />
+            <sp-field-label for="picker">Where do you live?</sp-field-label>
+            <sp-picker id="picker">
+                <span slot="label">
+                    Select a Country with a very long label, too long in fact
+                </span>
+                <sp-menu-item>Deselect</sp-menu-item>
+                <sp-menu-item>Select Inverse</sp-menu-item>
+                <sp-menu-item>Feather...</sp-menu-item>
+                <sp-menu-item>Select and Mask...</sp-menu-item>
+                <sp-menu-divider></sp-menu-divider>
+                <sp-menu-item>Save Selection</sp-menu-item>
+                <sp-menu-item disabled>Make Work Path</sp-menu-item>
+            </sp-picker>
+        </sp-theme>
         <script async src="main.bundle.js"></script>
     </body>
 </html>

--- a/projects/example-project-webpack/src/index.js
+++ b/projects/example-project-webpack/src/index.js
@@ -14,8 +14,11 @@ governing permissions and limitations under the License.
 // import './styles.css';
 
 // import the components we'll use in this page
-import '@spectrum-web-components/button/sp-button';
-import '@spectrum-web-components/field-label/sp-field-label';
-import '@spectrum-web-components/picker/sp-picker';
-import '@spectrum-web-components/menu/sp-menu';
-import '@spectrum-web-components/menu/sp-menu-item';
+import '@spectrum-web-components/theme/sp-theme.js';
+import '@spectrum-web-components/theme/scale-medium.js';
+import '@spectrum-web-components/theme/theme-light.js';
+import '@spectrum-web-components/button/sp-button.js';
+import '@spectrum-web-components/field-label/sp-field-label.js';
+import '@spectrum-web-components/picker/sp-picker.js';
+import '@spectrum-web-components/menu/sp-menu.js';
+import '@spectrum-web-components/menu/sp-menu-item.js';

